### PR TITLE
Add Private Internet Access

### LINKED ISSUE #5104: Private Internet Access
image

**Brand Name:** Private Internet Access
**Website:** https://www.privateinternetaccess.com/
**Alexa rank:** [~16k](https://www.alexa.com/siteinfo/privateinternetaccess.com)
  

**Official resources for icon and color:**

- https://www.privateinternetaccess.com/pages/press
  - There are vector versions, but it wont be necessarily easy to make it works here
- I would probably choose `#39b54a`, but I'm not entirely sure.

### PR DESCRIPTION
**Pull request by PeterShaggyNoble**
Private Internet Access

**Issue:** Resolves #5104
**Alexa rank:** [~16.2k](https://www.alexa.com/siteinfo/privateinternetaccess.com)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Icon & colour from [this SVG](https://www.privateinternetaccess.com/press/mascot/pia-mascot.svg).

### CODE REVIEW

**Review by mondeja — APPROVED** [review]
LGTM :+1: Thank you @PeterShaggyNoble :clap:
I'm going to wait for @service-paradis review before the merge since you've requested his review.

**Review by service-paradis — APPROVED** [review]
Good job, thanks @PeterShaggyNoble! 🔒 💯

**Pull request merged**
